actionpack/lib/action_controller/metal/redirecting.rb

```diff
@@ -71,23 +71,26 @@ def redirect_to(options = {}, response_status = {})
     # The referrer information is pulled from the HTTP +Referer+ (sic) header on
     # the request. This is an optional header and its presence on the request is
     # subject to browser security settings and user preferences. If the request
-    # is missing this header, the <tt>fallback_location</tt> will be used.
+    # is missing this header, the <tt>fallback_location</tt> or value of block will be used.
     #
-    #   redirect_back fallback_location: { action: "show", id: 5 }
-    #   redirect_back fallback_location: @post
+    #   redirect_back { { action: "show", id: 5 } }
```

Comment:
This form is not clean as having `fallback_location` option. Instead of having a block form let's only accept a proc in `fallback_location:` option,

Reply:
Hmmm... that's a good point. I think the block really shines when using a URL helper. You could theoretically do this by passing a proc too though so is that a valid reason to reject this shorthand?

Reply:
It's already accepting proc.

I used very similar helper, when there was no `redirect_back`. It was like `redirect_back_or_to { some_url }`. I don't like `fallback_location` option, because it forces passing not-lazy arguments, and it's also unnecessary long to write & read.

Closing?

Reply:
In this case eager arguments is not a problem. URL generation is not slow enough to force people to use lazy arguments, do instead of closing we could change the PR to document usage with proc.

Reply:
Ah, just saw that the documentation is already there, so yeah, we can close.